src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const ul = document.querySelector('ul');
+const listInfo = ul.querySelectorAll('li');
+// create a dataBase
+const dataBase = [];
+
+// add a data to dataBase
+for (const li of listInfo) {
+  dataBase.push(li);
+}
+
+// // create a convert function helper
+// function convert(string) {
+//   return parseFloat(string.replace(',', '.').slice(1));
+// }
+
+// upgrade 2.0
+function convert(employe) {
+  return parseFloat(employe.dataset.salary.replace(',', '.').slice(1));
+}
```

Comment:
The `convert` function assumes that each `li` element has a `data-salary` attribute. Ensure that this attribute exists on all `li` elements to prevent potential errors. You might want to add a check or handle cases where the attribute is missing.